### PR TITLE
Fixed broken URL

### LINKED ISSUE #1034: Page not found
**Description**

When getting invited to the community the link the bot is sending doesn't work

https://github.com/orgs/EddieJaoudeCommunity/people

I think the link should be

https://github.com/orgs/EddieHubCommunity/people

Raised a PR to fix this issue.https://github.com/EddieHubCommunity/support/pull/1036

### PR DESCRIPTION
**Pull request by CreaTorAlexander**
closes #1034 

#### What does this PR do?
Replaced the link where you can make the organisation public

### CODE REVIEW

**Review by SashoStoichkov — DISMISSED** [review]
Thank you for spotting 👍🏻 LGTM

**Comment by SashoStoichkov** [comment]
closes #1034

**Comment by CreaTorAlexander** [comment]
Oh didn't see. Have updated it 👍

File: .github/workflows/invitation.yml [hunk 1]
```diff
@@ -12,6 +12,6 @@ jobs:
...
+          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public so it appears on your GitHub profile for everyone else to see, you can do this by finding your name in the GitHub organisation list and change the dropdown to public https://github.com/orgs/EddieHubCommunity/people<br><br>Tips for practising:<br><br>Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo<br>Practice repo, instructions on how to add your name to the README in the README https://github.com/EddieJaoudeCommunity/hacktoberfest-practice<br>Remember contributing to open source is not just about code, its about collaboration, communication and adding value<br>I hope that helps'
```
Comment (Vyvy-vi) [inline .github/workflows/invitation.yml:15]:
```suggestion
          comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public so it appears on your GitHub profile for everyone else to see, you can do this by finding your name in the GitHub organisation list and change the dropdown to public https://github.com/orgs/EddieHubCommunity/people<br><br>Tips for practising:<br><br>Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo<br>Practice repo, instructions on how to add your name to the README in the README https://github.com/EddieHubCommunity/hacktoberfest-practice<br>Remember contributing to open source is not just about code, its about collaboration, communication and adding value<br>I hope that helps'
```
Can the link to the hacktoberfest-practice repo also be updated?

**Review by Vyvy-vi — CHANGES_REQUESTED** [review]

File: .github/workflows/invitation.yml (hunk 1 above)
Comment (SashoStoichkov) [reply .github/workflows/invitation.yml:15]: As always you spot everything 👀

**Review by Vyvy-vi — APPROVED** [review]
LGTM-!

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍 , thank you all 🤓

**Pull request merged**